src/ajax/parseXML.js

```diff
@@ -9,7 +9,8 @@ jQuery.parseXML = function( data ) {
 		return null;
 	}
 
-	// Support: IE9
+	// Support: IE 9-11 only
```

Comment:
I thought we discussed that a while ago at the meeting, it seems conclusion was to mention even those browsers that we do not support